torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2444,6 +2478,18 @@ def sample_inputs_masked_select(op_info, device, dtype, requires_grad):
                         dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16)),
            ),
            sample_inputs_func=sample_inputs_hstack_dstack_vstack),
+    OpInfo('unfold',
+           op=lambda x, *args: x.unfold(*args),
+           dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16),
+           test_inplace_grad=False,
+           check_batched_gradgrad=False,
```

Comment:
@zou3519 said this is a known issue and that these tests were being skipped in the method_tests too. could you delete the following entries now that we are porting the tests to `OpInfo`: https://github.com/pytorch/pytorch/blob/a06cf5d8a4184b5a5787f35d32b05dcc8747ed91/test/test_autograd.py#L5049-L5051